### DISCUSSION TITLE
posts/etc/sale-chair

### DISCUSSION THREAD

**Discussion by giscus** [movie42/hyunsu#13] [General]
# posts/etc/sale-chair

교회 소그룹에서 책 나눔을 하는데 그룹원 A가 말을 잇지 못하고 갑자기 눈물을 쏟았다. A는 직장 상사에게 괴롭힘을 당하고 있었다.
직종과 환경이 달라 차마 말을 건내지 못하고 ...

https://www.hyunsu.info/posts/etc/sale-chair

**Comment by hotbreakb** [comment]
두려워도 시도하는 것이 용기라는 말이 와닿네요.

저는 요즘 설득의 과정이 삶에서 빠지지 않았나 하는 생각이 들어요. 초등학교에서 다른 사람과 말하기를 배울 때 공감하기를 제일 먼저 배운다고 하죠. 그만큼 공감하는 게 중요하다는 걸 알고 있고, 저는 다른 사람의 상황에 공감하는 능력이 떨어져서 책을 읽고 있어요. 그런데 이게 치중되다 보니까, 다른 사람들의 말을 듣고 순응하는 자세가 늘어나서 제 의견을 피력하기 위해서 노력하는 일은 별로 없었던 거 같아요. 원래는 '이게 맞지 않나?'라고 생각했는데, 이제는 '그것도 맞을 수 있겠네.'라면서 이유를 물어보고 선택지를 A와 B 둘 중에 하나보다는 A와 B의 전체 집합 중 어떤 것이라고 생각을 하는 거죠. 이런 면에서 제가 너무 사회에 맞추려고 하는 건 아닌가 하는 생각도 드네요.

**Reply by movie42** [reply]
설득과 공감에 대한 이야기 일 수 도 있으나 어찌보면 내 자아와 사회가 요구하는 나를 두고 갈등하고 있는 고민 같아 보이기도 합니다. 참 어려운 문제인듯 합니다. 아마 누구도 완전하게 '딱 이거야'라고 답을 제시해주기도 어려울 것 같네요. 이건 죽을 때까지 노인이 되서도 하는 줄타기입니다.

하지만 단순하게 생각하면 사회속 나의 서열은 얼마나 되는가에 대한 이야기이기도 합니다. 설득은 아무리 논리적이어도 상대방이 듣고 싶어하지 않으면 설득이 안되는 것이고, 자아 없는 지속적인 공감과 수용은 노동입니다. 하지만 사회에서 내 랭킹이 높을 수록 설득과 공감이 필요 없어집니다. 이건 단순하게 생각했을 때 이야기입니다. 

독서를 많이 하신다니 좋은 것 같습니다.
